src/autoload/Multiplayer.gd

```diff
@@ -8,6 +8,10 @@ signal latency_updated
 const DEFAULT_NAME := "Guest"
 const DEFAULT_COLOR := Color8(255, 255, 255)
 
+# Free-for-all mode.
+const MODE_FFA := 0
+# Team battle mode.
+const MODE_TEAM := 1
```

Comment:
Replace with
```gdscript
enum {MODE_FFA, MODE_TEAM}
```
Or alternatively, something like
```gdscript
enum GameMode {FFA, TEAM}

# ...

var game_mode := GameMode.FFA
```

https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#enums

Reply:
It's a bit verbose now, I wish there was a way to do like `from Multiplayer import GameMode` or something. Plus type hinting doesn't work. Is this better in 4.0?